orbit/pkg/table/extension_kolide_logger.go

```diff
@@ -0,0 +1,37 @@
+package table
+
+import (
+	"fmt"
+	"github.com/rs/zerolog"
+	"github.com/rs/zerolog/log"
+)
+
+// Logger is a wrapper around zerolog, which we use for Kolide tables.
+// Kolide uses a go-kit logger, while we are using zerolog.
+type Logger struct {
+	zerolog.Logger
+}
+
+// Fatal logs a fatal message, just like Kolide's logutil.Fatal.
+func (l *Logger) Fatal(args ...interface{}) {
+	l.Logger.Fatal().Msg(fmt.Sprintln(args...))
+}
```

Comment:
I suspect this method may not be needed. We can double check once all the Kolide tables are in.